docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,448 @@
+# Initialize Vendor Accounts
+
+The DANDI ecosystem relies on vendor services to operate.  So first you will need to set up accounts with the following vendors:
+
+• **Heroku**
+
+• **AWS**
+
+• **GitHub**
+
+• **Terraform Cloud**
+
+• **Netlify**
+
+• **Sentry**
+
+• **PyPI**
```

Comment:
i think it would be good to add what each of these services provide to the infrastructure.

Reply:
Thanks Satra -- included a brief blurb for what each service is responsible for